deployer/aws-lambda/content-origin-request/index.js

```diff
@@ -1,7 +1,23 @@
 const sanitizeFilename = require("sanitize-filename");
+const { resolveFundamental } = require("@mdn/fundamental-redirects");
+const { DEFAULT_LOCALE, VALID_LOCALES } = require("@mdn/global-constants");
+const acceptLanguageParser = require("accept-language-parser");
 
 const CONTENT_DEVELOPMENT_DOMAIN = ".content.dev.mdn.mozit.cloud";
 
+const VALID_LOCALES_LIST = [...VALID_LOCALES.values()];
```

Comment:
I guess it's a matter of either doing this once, like this, and maybe never using it during the lifespan of this Lambda function, or creating and caching `[...VALID_LOCALES.values()]` the first time we call `guessLanguage`, or just  re-creating `[...VALID_LOCALES.values()]` each time we call `guessLanguage`. My guess is that during the lifespan of this Lambda function, which should be quite long given that it'll be called for every document request that wasn't served already from the CloudFront cache, we'll most likely use it at least once, and because of that I would say it's fine as it is.

Reply:
So Lambda@Edge will import the code and re-use it for multiple requests?

Reply:
@peterbe Yes, that's right!

Reply:
Ryan explained to me that it does. So initializing potentially expensive stuff in the module might be reusable across multiple URLs coming into the CF distribution.